wp-content/mu-plugins/pantheon/pantheon-cache.php

```diff
@@ -228,9 +228,9 @@ public function cache_admin_bar_render() {
 		if ( ! is_user_logged_in() )
 			return false;
 
-		if ( function_exists( 'current_user_can' ) && false == current_user_can( 'delete_others_posts' ) )
-			return false;
-
+		if ( function_exists( 'current_user_can' ) && !current_user_can( 'delete_others_posts' ) )
```

Comment:
Hi Daniel manage_options is much better, just tested it on client's site and works perfectly, thank you.